test/password_test.exs

```diff
@@ -0,0 +1,30 @@
+defmodule Fields.PasswordTest do
+  use ExUnit.Case
+  alias Fields.{Password, Helpers}
+
+  test ".type is :binary" do
+    assert Password.type() == :binary
+  end
+
+  test ".cast converts a value to a string" do
+    assert {:ok, "42"} == Password.cast(42)
+    assert {:ok, "atom"} == Password.cast(:atom)
+  end
+
+  test ".dump returns an Argon2id Hash given a password string" do
+    {:ok, result} = Password.dump("password")
+    assert is_binary(result)
+    assert String.starts_with?(result, "$argon2id$v=19$m=65536,t=6,p=1$")
```

Comment:
is this test dependent on the underlying version of the argon2 implementation? 
(_i.e. will it fail when the package is updated?_)

Reply:
Yeah it looks like it will. We could change it to:
```suggestion
    assert String.starts_with?(result, "$argon2")
```